x/tally/keeper/endblock.go

```diff
@@ -199,25 +206,45 @@ func (k Keeper) FilterAndTally(ctx sdk.Context, req types.Request, params types.
 		sort.Strings(reveals[i].ProxyPubKeys)
 	}
 
-	// Phase I: Filtering
+	// Phase 1: Filtering
 	filterResult, filterErr := ExecuteFilter(reveals, req.ConsensusFilter, req.ReplicationFactor, params)
-	result.Consensus = filterResult.Consensus
-	result.ProxyPubKeys = filterResult.ProxyPubKeys
-	result.TallyGasUsed += filterResult.GasUsed
+	// Begin tracking tally result and burn amount.
+	result := TallyResult{
+		Consensus:    filterResult.Consensus,
+		ProxyPubKeys: filterResult.ProxyPubKeys,
+		TallyGasUsed: filterResult.GasUsed,
+	}
+	burnAmt := math.ZeroInt()
```

Comment:
I think in phase 1 we don't have to consider the amount burned yet and we can simply deal with gas units. Only once we get ready to send it to the contract do we convert to `aseda`.